cognite/client/data_classes/datapoints.py

```diff
@@ -799,9 +799,8 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
                 or not len(self.status_symbol) == len(datapoints) == len(self.status_code)
             ):
                 raise ValueError("The number of status codes/symbols does not match the number of datapoints")
-
-            for dp, code, symbol in zip(datapoints, self.status_code, self.status_symbol):
-                dp["status"] = {"code": code, "symbol": symbol}
+            for dp in datapoints:
+                dp["status"] = {"code": dp.pop("statusCode"), "symbol": dp.pop("statusSymbol")}
```

Comment:
You are popping off the `statusCode` and `statusSymbol` fields because they are still kept in the `self.status_code` and `self.status_symbol` fields?

Reply:
To avoid the dump having both "status" (the object), "statusCode" and "statusSymbol". Just an oversight from previously.